### ISSUE TITLE
gvr-accessibility doesn't work properly anymore

### ISSUE THREAD

**Issue by liaxim** [gearvrf/GearVRf-Demos#265]
I guess it needs updates to work with the latest framework. All I see is a piedestal with a book - can't get to the accessibility menu that is supposed to be there when you look down. There was more to that sample than the piedestal and the book as far as I can recall.

**Comment by rahul27** [comment]
I think it has something to do with missing textures. 

Would see this occasionally when I run the app. Does it happen every time you launch the app?

**Comment by liaxim** [comment]
Launched maybe 2-3 times but it was always the same result.

**Comment by liaxim** [comment]
gvr-accessibility has been working fine for some time.
